app-modules/notifications/src/Listeners/NotifyUserOfSubscriptionCreated.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Assist\Notifications\Listeners;
+
+use Filament\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Assist\Notifications\Events\SubscriptionCreated;
+
+class NotifyUserOfSubscriptionCreated implements ShouldQueue
+{
+    public function handle(SubscriptionCreated $event): void
+    {
+        Notification::make()
+            ->status('success')
+            ->title("You have been subscribed to {$event->subscription->subscribable->full}")
+            ->sendToDatabase($event->subscription->user);
```

Comment:
Do you think we might also want to deliver these via mail? 

An idea to consider for some flexibility: Standard Laravel notifications via `mail()` and `toArray()` channels. Send the DB notification immediately, delay the mail notification. If the user has read the DB notification, don't send the mail when we run a check after the delay has elapsed.

Reply:
Hmm, interesting. We could want to do that at some point.

Is it possible to wrap the logic into a Laravel Notification that could trigger either an email or Filament Notification? That way we have notifications in a central place.

Reply:
Yes - I think we'd be able to do that by just adding the `toDatabase()` method in our Laravel notification, and dispatching the Filament notification within that method.